rust/src/router/plain_transport.rs

```diff
@@ -683,6 +655,19 @@ impl PlainTransport {
             let data = Arc::clone(&data);
 
             channel.subscribe_to_fbs_notifications(id.into(), move |notification| {
+                let Ok(Some(notification::BodyRef::PlainTransport(notification_wrapper))) =
+                    notification.body()
+                else {
+                    panic!("Wrong message from worker: {notification:?}");
+                };
+
+                // Get the Notification instance.
+                let notification = notification_wrapper
+                    .notification()
+                    .unwrap()
+                    .try_into()
+                    .unwrap();
```

Comment:
Why is `try_into` needed here? Reading just on GitHub this looks a bit confusing.

Reply:
This is getting a `Notification` object out a `NotificationRef`. Otherwise each field in  `NotificationRef` is accessed via a method with the field name, which returns a result with the value. IMHO it's too cluttered.

Reply:
NOTE: We will finally use `NotificationRef` instead as noted [here](https://github.com/versatica/mediasoup/pull/1194)